ownCloud/Client/Actions/Actions+Extensions/OpenInAction.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  OpenInAction.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 12/11/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import ownCloudSDK
+
+class OpenInAction: Action {
+	override class var identifier : OCExtensionIdentifier? { return OCExtensionIdentifier("com.owncloud.action.openin") }
+	override class var category : ActionCategory? { return .normal }
+	override class var name : String { return "Open in".localized }
+
+	override class func applicablePosition(forContext: ActionContext) -> ActionPosition {
+		if forContext.items[0].type == .collection {
```

Comment:
As mentioned in a previous comment, to make `Action`s truly universally usable throughout the app, it should look at all `items`. In this case, if any of the items is a `.collection`, that'd mean this action isn't available.

Reply:
I think it depends on the action itself. Of course, a `Delete` action should delete as many items as you want, but for open in, this action should not be available in multiselection environments because of `UIDocumentInteractionController` can be only created with one `URL` and I think we should not open more than one `UIDocumentInteractionController`. Same for the `Rename` action.